subsys/cpp/include/cstdint

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file
+ *
+ * @brief Stub header allowing compilation of `#include <cstdint>`
+ */
+
+/*
+ * Copyright 2021 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Nit: It would be nice to keep the order of these header comments consistent (for instance, `cstddef` begins with the copyright header followed by the Doxygen header).